nflow-rest-api-jax-rs/src/test/java/io/nflow/rest/v1/jaxrs/WorkflowDefinitionResourceTest.java

```diff
@@ -116,7 +115,7 @@ public void listWorkflowDefinitionsFindsExistingAndStoredDefinitionsWithDbType()
 
   private List<ListWorkflowDefinitionResponse> getDefinitionListType(List<String> list) {
     try (Response r = resource.listWorkflowDefinitions(list)) {
-      return r.readEntity(definitionListType);
+      return asList(r.readEntity(ListWorkflowDefinitionResponse[].class));
```

Comment:
Why this change? First read as array. Then copy it into a list rather than directly add items to a list - not that there will be so many definitions that it the performance difference would matter.

Reply:
Just to get rid of the ugly GenericType definitions. Anyway this is just test code. Maybe this method could return an array?

Reply:
Reverted